install/jms/bin/ts.jte

```diff
@@ -210,39 +210,39 @@ tslib.name=jmstck
 
 #########################################################################
 # platform.mode is used by to specify runtime environment for testing
-# which can be either javaEE or standalone. For JMS TCK this setting
+# which can be either javaEE or standalone. For Jakarta Messaging TCK this setting
```

Comment:
the **platform.mode** remains _javaEE_?

Reply:
https://github.com/eclipse-ee4j/jakartaee-tck/pull/417 has a conversation started about whether ```platform.mode``` possible values should be updated in EE 9 or if we are out of time for changes like that.